src/registrar/assets/src/js/getgov-admin/helpers-portfolio-dynamic-fields.js

```diff
@@ -469,11 +470,36 @@ export function handlePortfolioSelection(
             if (requestedSuborganizationField) showElement(requestedSuborganizationField);
             if (suborganizationCity) showElement(suborganizationCity);
             if (suborganizationStateTerritory) showElement(suborganizationStateTerritory);
+            
+            // Handle rejectSuborganizationButtonFieldset (display of the clear requested suborg button).
+            // Basically, this button should only be visible when we have data for suborg, city, and state_territory.
+            // The function handleSuborgFieldsAndButtons() in domain-request-form.js handles doing this same logic
+            // but on field input for city, state_territory, and the suborg field.
+            // If it doesn't exist, don't do anything.
+            if (!rejectSuborganizationButtonFieldset){
+                console.warn("updateSuborganizationFieldsDisplay() => Could not update rejectSuborganizationButtonFieldset")
+                return;
+            }
+
+            // Initially show / hide the clear button only if there is data to clear
+            let requestedSuborganizationFieldInput = document.getElementById("id_requested_suborganization");
+            let suborganizationCityInput = document.getElementById("id_suborganization_city");
+            let suborganizationStateTerritoryInput = document.getElementById("id_suborganization_state_territory");
```

Comment:
Initialize these consts in the handlePortfolioSelection initialization alongside related consts (requestedSuborganizationField, suborganizationCity, and suborganizationStateTerritory)